### DISCUSSION TITLE
Selectively assign 'mode' in uncontrolled component

### DISCUSSION THREAD

**Discussion by danbergelt** [react-hook-form/react-hook-form#3326] [Q&A] [answered]
Hello,

I have a form where I would like to selectively assign the mode to `onChange` for some inputs, and selectively assign the mode to `onSubmit` for other fields. Is it possible to do this at the input level, or do these rules only apply at the form level?

**Comment by bluebill1049** [comment, marked as answer]
set mode to `onSubmit`and trigger validation with your own rules.

**Comment by danbergelt** [comment]
Yep that was a super easy fix, I'm calling `trigger` onChange for the desired fields. Thank you for the quick reply and the great library.

**Reply by bluebill1049** [reply]
np
